kivy_ios/toolchain.py

```diff
@@ -209,8 +209,17 @@ def get_env(self):
                  'include_file_mtime,include_file_ctime,file_stat_matches'))
 
         if not self._ccsh:
-            self._ccsh = tempfile.NamedTemporaryFile()
-            self._cxxsh = tempfile.NamedTemporaryFile()
+            def noicctempfile():
+                '''
+                reported issue where C Python has issues with 'icc' in the compiler path
+                '''
```

Comment:
Can you reference the upstream bug in the docstring.
That way it's easier to figure out if the hack can be removed when we come back to it later.
https://github.com/python/cpython/pull/96399